MongoBundle/Resources/config/transformer.yml

```diff
@@ -1,2 +1,7 @@
 parameters:
     open_orchestra.transformer.boolean_to_bdd.class: OpenOrchestra\Transformer\ConditionFromBooleanToMongoTransformer
+services:
+    open_orchestra.transformer.boolean_to_bdd:
+        class: %open_orchestra.transformer.boolean_to_bdd.class%
+        arguments:
+            - ''
```

Comment:
Can't this be a default value in the contructor ? It sounds weird to pass an empty parameter in the conf.